src/libostree/ostree-repo-private.h

```diff
@@ -152,6 +152,7 @@ struct OstreeRepo {
   gid_t target_owner_gid;
   guint min_free_space_percent; /* See the min-free-space-percent config option */
   guint64 min_free_space_mb; /* See the min-free-space-size config option */
+  gboolean emergency_cleanup;
```

Comment:
Bikeshed: how about `cleanup_stagedir`? The term "emergency" there is not very precise.

Reply:
Addressed the issue in a following fixup! commit. 
Thank you for the review! :+1: